scripts/monitoring/auto_scale_workers.py

```diff
@@ -64,15 +64,19 @@ def get_queue_length(queue):
     # https://github.com/boto/boto3/issues/599
     # Ref: https://boto3.amazonaws.com/v1/documentation/api/latest/guide/sqs.html#using-an-existing-queue
     # Ref: https://boto3.amazonaws.com/v1/documentation/api/latest/reference/services/sqs.html#SQS.Queue.attributes
-    return (
-        queue.attributes["ApproximateNumberOfMessages"]
-        + queue.attributes["ApproximateNumberOfMessagesNotVisible"]
+    return int(queue.attributes["ApproximateNumberOfMessages"]) + int(
+        queue.attributes["ApproximateNumberOfMessagesNotVisible"]
     )
 
 
 def get_queue_length_by_challenge(challenge):
     queue_name = challenge["queue"]
-    queue = get_sqs_queue_by_name(queue_name)
+    try:
+        queue = get_sqs_queue_by_name(queue_name)
+    except Exception as _e:  # noqa: F841
+        raise Exception(
+            "Unable to fetch queue with name: {}. Skipping.".format(queue_name)
```

Comment:
If you are handling the exception here you should add the raise exception on L57